libs/damap/src/lib/components/dmp/summary/summary.component.html

```diff
@@ -9,7 +9,9 @@
     <th mat-header-cell *matHeaderCellDef>
       {{ "dmp.steps.summary.table.header.step" | translate }}
     </th>
-    <td mat-cell *matCellDef="let element">{{ element.step | translate }}</td>
+    <td mat-cell *matCellDef="let element; let i = index">
+      <a (click)="navigateToStep(i)">{{ element.step | translate }}</a>
```

Comment:
Would it make sense to add a short tooltip and an aria label for accessibility on the `a` element?
Something like
```
        matTooltip="Jump to section"
        aria-label="Link to jump to section `xyz` and displays a tooltip when focused or hovered over"```

Reply:
Done